components/automate-ui/src/app/pages/+compliance/+credentials/components/credentials-list.html

```diff
@@ -35,7 +35,7 @@
                   Last Modified
                   <chef-sort-toggle sort="last_modified" [order]="orderFor('last_modified')"></chef-sort-toggle>
                 </chef-th>
-                <chef-th></chef-th>
+                <chef-th class="three-dot-column"></chef-th>
```

Comment:
applying this class matches the new width changes of the control menu cells that were just made.